lib/api/include/irods/simpleQuery.h

```diff
@@ -25,8 +25,12 @@ typedef struct {
 #ifdef __cplusplus
 extern "C"
 #endif
-    int
-    rcSimpleQuery(rcComm_t* conn, simpleQueryInp_t* simpleQueryInp, simpleQueryOut_t** simpleQueryOut)
-        __attribute__((deprecated("SimpleQuery is deprecated. Use GenQuery or SpecificQuery instead.")));
+
+    // clang-format off
+
+    __attribute__((deprecated("SimpleQuery is deprecated. Use GenQuery or SpecificQuery instead.")))
+    int rcSimpleQuery(rcComm_t* conn, simpleQueryInp_t* simpleQueryInp, simpleQueryOut_t** simpleQueryOut);
+
+    // clang-format on
```

Comment:
Another formatting note... not sure why clang-format only wants one of these comments to be indented (namely, the one on 29). I think it should be both or neither.

Reply:
Let's ignore indentation due to `extern "C"` here.

I really hate that clang-format wants to indent the function declaration.

Reply:
Okay. In that case, I'm just going to remove the clang-format off/on comments and ignore clang-format :p